events/events_network_selectstats.c

```diff
@@ -88,12 +88,12 @@ events_network_selectstats_select(void)
  * function was called.
  */
 void
-events_network_selectstats(double * _N, double * _mu, double * _va,
+events_network_selectstats(double * arg_N, double * _mu, double * _va,
```

Comment:
I also couldn't pick whether I should make a minimal change here (only modifying `_N`) or make a consistent change (also modifying `_mu`, `_va`, and `_max` as well, even though those aren't reserved identifiers).